### DISCUSSION TITLE
Screen orientation

### DISCUSSION THREAD

**Discussion by steelcd** [nebhead/PiFire#61] [Q&A] [answered]
Hello,

I'm using an ILI9341 screen from HiLetgo.  I need to rotate the screen 180 degrees.  What syntax should I use in the display_ili9341.py file to make it flip?

Thanks!

Chris

**Comment by steelcd** [comment, marked as answer]
Found it!

Add a rotate parameter and set it to two:

self.device = ili9341(self.serial, active_low=False, width=self.WIDTH, height=self.HEIGHT, gpio_LIGHT=26, rotate=2)

**Reply by nebhead** [reply]
Hey @steelcd!  Sorry I missed this, as I was out of town for a few days.  I think you got the answer you needed though, so that's great!  Let me know if you need any other help.
